README.md

```diff
@@ -293,4 +294,11 @@ At the `resize` URL (e.g. `https://example.com/resizeiframe.html`), you should s
 </script>
 ```
 
+## Third-party Libraries
 
+Some developers have kindly contributed Posting API clients for different programming languages. If you know something that is missing from this page, please [create a pull request](https://github.com/lever/postings-api/pull/new/master).
+
+Please note that the software is not endorsed or certified by Lever.
+
+### PHP
+* <https://github.com/mjacobus/lever-api-client>
```

Comment:
This looks like non-standard markdown. I think you might want something closer to:

```md
* [mjacobus/lever-api-client](https://github.com/mjacobus/lever-api-client)
```